commission/models/commission_mixin.py

```diff
@@ -23,8 +23,7 @@ class CommissionMixin(models.AbstractModel):
     product_id = fields.Many2one(comodel_name="product.product", string="Product")
     commission_free = fields.Boolean(
         string="Comm. free",
-        related="product_id.commission_free",
-        store=True,
+        default=lambda m: m.product_id.commission_free,
```

Comment:
This is not valid, and it may work by chance. Change to a computed field if the related is not initialized on time.

Reply:
Changed to compute